src/app/config/or_options_st.h

```diff
@@ -1103,6 +1103,12 @@ struct or_options_t {
    **/
   int DormantCanceledByStartup;
 
+  /** Boolean: true if we should export rend point when exporting CircuitID. */
+  int HiddenServiceExportRendPoint;
+
+  /** Instance ID to export when exporting CircuitID. */
+  int HiddenServiceExportInstanceID;
```

Comment:
Documenting the Min, Max and Default here would be desirable.

Reply:
Fixed in `ec8f17d`.